python/python/Ice/__init__.py

```diff
@@ -1948,7 +1948,6 @@ def _callbackOnInterruptCallback(self, sig):
 
 Object._op_ice_isA = IcePy.Operation(
     "ice_isA",
-    OperationMode.Idempotent,
     OperationMode.Nonmutating,
```

Comment:
It looks like you removed the wrong one.

Reply:
This was intentional. Maybe this is room for potential cleanup...
but in every language, we go out of our way to use `OperationMode::Nonmutating` when implementing the core set of `Object` operations. Each language even has tests which override the Object operations, _just_ to check that they're using `Nonmutating`.

Swift:
https://github.com/zeroc-ice/ice/blob/646965d371b15d6935aa60e3dec248b45bab166a/swift/src/Ice/Proxy.swift#L400-L404 C++:
https://github.com/zeroc-ice/ice/blob/646965d371b15d6935aa60e3dec248b45bab166a/cpp/src/Ice/ProxyAsync.cpp#L330-L337 C#:
https://github.com/zeroc-ice/ice/blob/646965d371b15d6935aa60e3dec248b45bab166a/csharp/src/Ice/Proxy.cs#L633-L635 you get the idea...

Isn't this necessary for backwards compatibility? Since we strictly check that the operation mode matches?

Reply:
The whole point of this exercise is to eliminate Nonmutating everywhere.

See #370.

Reply:
Alright, I see. This PR was focused on removing the metadata, and `sendMode`, since without this metadata, it's redundant. I'll remove these uses in a follow-up PR.
#370 really only covered the metadata.